ENVIRONMENT.rst

```diff
@@ -65,6 +65,9 @@ Environment Configuration Settings
 - **SWIFT_PROJECT_DOMAIN_NAME**:
 - **SWIFT_PROJECT_DOMAIN_ID**:
 - **WALE_SWIFT_PREFIX**: (optional) the full path to the backup location on the Swift Storage in the format swift://bucket-name/very/long/path. If not specified Spilo will generate it from WAL_SWIFT_BUCKET.
+- **WALG_AZ_PREFIX**: (optional) the azure prefix to store WAL backups at in the format azure://test-container/walg-folder.
+- **AZURE_STORAGE_ACCOUNT**:(optional) the azure storage account to use for WAL backups.
+- **AZURE_STORAGE_ACCONT_ACCESS_KEY**:(optional) the access key for the azure storage account used for WAL backups.
```

Comment:
There's a typo here:
```suggestion
- **AZURE_STORAGE_ACCOUNT_ACCESS_KEY**:(optional) the access key for the azure storage account used for WAL backups.
```

But I see later in the code the mention of `AZURE_STORAGE_ACCESS_KEY`, so I assume the environment variable should be this instead?